static/app/components/replays/replayTagsTableRow.spec.tsx

```diff
@@ -0,0 +1,93 @@
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+import {OrganizationContext} from 'sentry/views/organizationContext';
+import {RouteContext} from 'sentry/views/routeContext';
+import ReplayTagsTableRow from './replayTagsTableRow';
+
+const tags = [
+  {
+    key: 'foo',
+    value: ['bar', 'baz'],
+  },
+  {
+    key: 'release',
+    value: ['1.0.0', '2.0.0'],
+  },
+];
```

Comment:
Can we give these two sets of test data specific names?

saying `tags[1]` isn't as descriptive as something like `releaseTag` or `tagTestData['release']` or whatever other name scheme we could use.